pkg/apis/ceph.rook.io/v1/zz_generated.deepcopy.go

```diff
@@ -21,7 +21,7 @@ limitations under the License.
 package v1
 
 import (
-	v1alpha2 "github.com/rook/rook/pkg/apis/rook.io/v1alpha2"
+	rookiov1 "github.com/rook/rook/pkg/apis/rook.io/v1"
```

Comment:
Why use `rookiov1` here but `rookv1` elsewhere?

Reply:
I'll change all of them to rookv1. I started with rookiov1 in a few places, but missed making them consistent after I switched.

Reply:
Actually, this is just generated code so I can't change it.